less/mixins.less

```diff
@@ -1 +1,42 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.flex-align (@dir: column, @align: center) {
+    display: flex;
+    flex-direction: @dir;
+    align-items: @align;
+}
+
+.flex-just (@dir: row, @just: center) {
+    display: flex;
+    flex-direction: @dir;
+    justify-content: @just;
+}
+
+.base-button (@bgcolor, @fontcolor) {
+    border-radius: 10px;
+    width: 30%;
+    border: none;
+    height: 40px;
+    font-size: 1.6rem;
+    background: @bgcolor;
+    color: @fontcolor;
+
+    &:hover {
+        background: @porcelain;
+        color: @btn-bg-color;
+        .spinny(360deg, 3s);
```

Comment:
Very impressive way to get animations working on desktop and mobile. I never really played around with animations so thanks for teaching me this!